### ISSUE TITLE
How to test the trained model like MinUNet5cm

### ISSUE THREAD

**Issue by DiegoWangSys** [mit-han-lab/e3d#4]
Dear Ken, thanks for your great code! I have trained my MinkUNet on SemanticKitti and got checkpoints.pt file! Now I wanna test the model on 11~21 scenes and hope to get the predicted label, what should I do? I found a trigger-like code in spvnas/core/datasets/semantic_kitti.py:
68:         submit_to_server = kwargs.get("submit",False)

Should I change "False" to "True" and I will get the label on test set?

**Comment by kentangSJTU** [comment]
Hi,

Yes. When you modify the submit flag to "true", you may modify our [spvnas/evaluate.py](spvnas/evaluate.py) to get the predictions on the test set. Commenting line 64-71 and uncommenting line 73 will help you load your pretrained checkpoint.

By the way, we'll be releasing our code for test submission soon. Feel free to comment if there're questions before our code release. Thanks!


Best,
Haotian

**Comment by DiegoWangSys** [comment]
Dear Haotian, I tried what you told before but I am confusing that 
73    model = builder.make_model()

seems to construct a model from scratch. How did the code load the checkpoint (.pt file) and model parameters?

**Comment by kentangSJTU** [comment]
Hi,

This is managed by `torchpack`'s `SaverRestore` callback.

Best,
Haotian

**Comment by DiegoWangSys** [comment]
Thanks a lot, I get the prediction labels ,
